src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2172,6 +2181,16 @@ public function edit($intId=null, $ajaxId=null)
 							   ->execute(time(), $this->intId);
 			}
 
+			$arrRecords = $objSessionBag->get('edit_records', array());
+			$intLatestVersion = $objVersions->getLatestVersion();
+
+			// Show a warning if the record has been saved by another user (see #8412)
+			if ($intLatestVersion !== null && isset($arrRecords[$this->strTable][$this->intId]) && $intLatestVersion > ($arrRecords[$this->strTable][$this->intId] + 1))
+			{
+				\Message::addError(sprintf($GLOBALS['TL_LANG']['ERR']['versionWarning'], ($intLatestVersion - 1), $arrRecords[$this->strTable][$this->intId]));
+				$this->reload();
```

Comment:
When I read the code, the process is as follows: If somebody saved a new version until I hit the save button, it will reload and show the message but it will not store my data to the database. Is that correct?